cmd/gazelle/diff_test.go

```diff
@@ -105,12 +105,17 @@ func TestDiffNew(t *testing.T) {
 func TestDiffEmpty(t *testing.T) {
 	files := []testtools.FileSpec{
 		{Path: "WORKSPACE"},
+		{Path: "BUILD.bazel"},
 	}
 	dir, cleanup := testtools.CreateFiles(t, files)
 	defer cleanup()
 
 	if err := runGazelle(dir, []string{"-go_prefix=example.com/hello", "-mode=diff", "-patch=p"}); err != nil {
-		t.Fatalf("Expected no error, but got %q", err)
+		t.Error("Expected no diff, but got a diff.")
+		want := append(files, testtools.FileSpec{
```

Comment:
This code only checks that `p` is empty if `runGazelle` returns an error, but we want no error *and* and empty patch.

```
if err := runGazelle(...); err != nil {
  t.Fatalf("Expected no error, but got %q", err)
}
testtools.CheckFiles(t, dir, []testtools.FileSpec{{Path: "p"}})
```